UNRELEASED_CHANGELOG.md

```diff
@@ -40,7 +41,7 @@ It is not possible to remove users from distinct channels anymore.
 - Deprecated `ChannelController::unBanUser`, the `ChannelController::unbanUser` method should be used instead
 
 ### ❌ Removed
-
+- Removed `infinite` field from channel config
```

Comment:
So, we're mentioning it after all? PR description said we're not mentioning it on purpose 😄

Reply:
Ultimately removed this line from the changelog. We don't want to advertise that we make such careless mistakes.